llama_stack/cli/stack/_build.py

```diff
@@ -143,7 +143,7 @@ def run_stack_build_command(args: argparse.Namespace) -> None:
                 completer=WordCompleter(available_providers),
                 complete_while_typing=True,
                 validator=Validator.from_callable(
-                    lambda x: x in available_providers,
+                    lambda x, available_providers=available_providers: x in available_providers,
```

Comment:
not sure why this is needed...

Reply:
Function definition does not bind loop variable `available_providers` - RuffB023

Reply:
Yeah I get that ruff complains but it looks like a bug in ruff... e.g. this one https://github.com/astral-sh/ruff/issues/7847 ? I don't think we should bend for the tool if it generates false positives.